providers/universal-provider/test/shared/WalletClient.ts

```diff
@@ -268,6 +268,14 @@ export class WalletClient {
               );
               result = signedDirect.signature;
               break;
+            case "tezos_send":
+              //  eslint-disable-next-line no-case-declarations
+              const send_res = await this.tezosWallet.send(
```

Comment:
where is this.tezoswallet defined?

Reply:
we have this.cosmosWallet defined in line 29 in this file. But no tezos wallet defined.

Reply:
the unit tests are broken for UniversalProvider. They are hanging on init and after several hours of debugging I decided to just leave them as is. No need to make Tezos more covered than any other blockchain. Manual testing passed with example dapp